pkg/services/live/live.go

```diff
@@ -171,16 +219,71 @@ func InitalizeBroker() (*GrafanaLive, error) {
 		// Unknown path
 		ctx.Resp.WriteHeader(404)
 	}
-	return b, nil
+	return glive, nil
 }
 
-// Publish sends the data to the channel
-func (b *GrafanaLive) Publish(channel string, data []byte) bool {
-	_, err := b.node.Publish(channel, data)
+// GetChannelHandler gives threadsafe access to the channel
+func (g *GrafanaLive) GetChannelHandler(channel string) (models.ChannelHandler, error) {
```

Comment:
@marefr -- can you audit this function?  It is called frequently and potentially from multiple threads.